### PR TITLE
DOC-12347 Update rolling back migration

### PR DESCRIPTION
**Pull request by rao-shwe**
First draft
Added prerequisites for performing migration rollback operation from magma to couchstore.

Formatting is pending.

### CODE REVIEW

File: modules/manage/pages/manage-buckets/migrate-bucket.adoc [hunk 1]
```diff
@@ -184,4 +211,4 @@ include::manage:example$migrate-bucket-storage-backend.sh[tag=recover-node]
 include::manage:example$migrate-bucket-storage-backend.sh[tag=rebalance-cluster]
 ----
 
-. Repeat the previous step until all nodes that you'd migrated have rolled back to their original storage backend. 
+. Repeat the previous step until all nodes that you earlier migrated have rolled back to their original storage backend.
```
Comment (ggray-cb) [inline modules/manage/pages/manage-buckets/migrate-bucket.adoc]: While I'm not a strict about split infinitives, "earlier migrated" sounds a bit awkward. Maybe reword to "migrated earlier". Also could make the sentence more active: "Repeat the previous step until you have rolled back all of the nodes to their original storage backend."

**Review by ggray-cb — APPROVED** [review]
One minor suggested change. Otherwise, it's ready to go. Thanks!

File: modules/manage/pages/manage-buckets/migrate-bucket.adoc (hunk 1 above)
Comment (rao-shwe) [reply modules/manage/pages/manage-buckets/migrate-bucket.adoc]: I've made a minor change in your feedback and wrote the following:
"Repeat the previous step until you have rolled back all of the migrated nodes to their original storage backend."

Added the word "migrated".
